packages/usa-tooltip/src/test/test-patterns/test-usa-tooltip-utilities.twig

```diff
@@ -116,3 +116,15 @@
     <button type="button" class="{{ tooltip.classes }}" data-position="right" title="A longer Top tooltip that is longer than appx 320px" data-classes="{{ tooltip.util_classes }}" title="">Show on Right</button>
   </div>
 </div>
+<h2>With trigger positioning offsets</h2>
+<hr/>
+<p>In this scenario:</p>
+<ul>
+  <li>The tooltip should be positioned accounting for any offsets that might apply from the trigger</li>
+</ul>
+<button type="button" class="{{ tooltip.classes }} margin-top-1" data-position="top" data-classes="{{ tooltip.util_classes }}" title="Top">Show on top</button>
+<button type="button" class="{{ tooltip.classes }} margin-top-2" data-position="top" data-classes="{{ tooltip.util_classes }}" title="Top">Show on top</button>
+<button type="button" class="{{ tooltip.classes }} margin-top-3" data-position="top" data-classes="{{ tooltip.util_classes }}" title="Top">Show on top</button>
+<button type="button" class="{{ tooltip.classes }} margin-top-4" data-position="top" data-classes="{{ tooltip.util_classes }}" title="Top">Show on top</button>
+<button type="button" class="{{ tooltip.classes }} margin-top-5" data-position="top" data-classes="{{ tooltip.util_classes }}" title="Top">Show on top</button>
+<button type="button" class="{{ tooltip.classes }} margin-top-6" data-position="top" data-classes="{{ tooltip.util_classes }}" title="Top">Show on top</button>
```

Comment:
For simpler testing/review you could use the utility name as the text. _It wasn't immediately clear to me until I looked at the utils what was supposed to happen._

```suggestion
<button type="button" class="{{ tooltip.classes }} margin-top-1" data-position="top" data-classes="{{ tooltip.util_classes }}" title="Top">Margin top 1</button>
<button type="button" class="{{ tooltip.classes }} margin-top-2" data-position="top" data-classes="{{ tooltip.util_classes }}" title="Top">Margin top 2</button>
<button type="button" class="{{ tooltip.classes }} margin-top-3" data-position="top" data-classes="{{ tooltip.util_classes }}" title="Top">Margin top 3</button>
<button type="button" class="{{ tooltip.classes }} margin-top-4" data-position="top" data-classes="{{ tooltip.util_classes }}" title="Top">Margin top 4</button>
<button type="button" class="{{ tooltip.classes }} margin-top-5" data-position="top" data-classes="{{ tooltip.util_classes }}" title="Top">Margin top 5</button>
<button type="button" class="{{ tooltip.classes }} margin-top-6" data-position="top" data-classes="{{ tooltip.util_classes }}" title="Top">Margin top 6</button>
```

Reply:
> For simpler testing/review you could use the utility name as the text. _It wasn't immediately clear to me until I looked at the utils what was supposed to happen._

I incorporated this into the new examples in ddc53f0.